service/dbfs/handle.go

```diff
@@ -0,0 +1,225 @@
+package dbfs
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/databricks/databricks-sdk-go/useragent"
+)
+
+// DbfsFileMode conveys user intent when opening a file.
+type DbfsFileMode int
```

Comment:
remove `Dbfs` prefix, it's redundant because of the package name

Reply:
Done.